src/Zynga/Framework/IO/Disk/V1/Exception/FailedToWriteToFileException.hh

```diff
@@ -0,0 +1,10 @@
+<?hh // strict
+
+namespace Zynga\Framework\IO\Disk\V1\Exception;
+
+use Zynga\Framework\IO\Disk\V1\Exception\DiskIOException;
+
+/**
+ * Represents that an operation to write to a file failed
```

Comment:
These comments are nice but I feel the class name is indicative enough of what the error is. Your class names are 2good4me